ms/get.go

```diff
@@ -6,9 +6,9 @@ import (
 )
 
 // Get returns the value of a key, or null if the key doesn’t exist.
-func (ms Ms) Get(key string, options types.QueryOptions) (interface{}, error) {
+func (ms Ms) Get(key string, options types.QueryOptions) (string, error) {
```

Comment:
On redis side, both `key` and `value` can be an empty string. 

While the key case is maybe a bit far-fetched, I would use a pointer to a string as return type to distinct empty string result from `nil`

Reply:
Strings can never be `nil`

Reply:
As seen together: I misunderstood your comment and forgot that Redis never returns errors. So, yes, for that specific command, we should return a pointer on string, because Redis can either return `null` if the key does not exist, or `""` if it exists and is empty, which is an entirely different information.

:+1: